kclvm/tools/src/LSP/src/quick_fix.rs

```diff
@@ -1,17 +1,42 @@
 use std::collections::HashMap;
 
-use kclvm_error::{DiagnosticId, WarningKind};
+use kclvm_error::{DiagnosticId, ErrorKind, WarningKind};
 use lsp_types::{
     CodeAction, CodeActionKind, CodeActionOrCommand, Diagnostic, NumberOrString, TextEdit, Url,
 };
+use serde_json::Value;
 
 pub(crate) fn quick_fix(uri: &Url, diags: &Vec<Diagnostic>) -> Vec<lsp_types::CodeActionOrCommand> {
     let mut code_actions: Vec<lsp_types::CodeActionOrCommand> = vec![];
     for diag in diags {
         if let Some(code) = &diag.code {
             if let Some(id) = conver_code_to_kcl_diag_id(code) {
                 match id {
-                    DiagnosticId::Error(_) => continue,
+                    DiagnosticId::Error(error) => match error {
+                        ErrorKind::CompileError => {
+                            let replacement_text = extract_suggested_replacement(&diag.data)
+                                .unwrap_or_else(|| "".to_string());
+                            let mut changes = HashMap::new();
+                            changes.insert(
+                                uri.clone(),
+                                vec![TextEdit {
+                                    range: diag.range,
+                                    new_text: replacement_text,
+                                }],
+                            );
+                            code_actions.push(CodeActionOrCommand::CodeAction(CodeAction {
+                                title: ErrorKind::CompileError.name(),
```

Comment:
![image](https://github.com/kcl-lang/kcl/assets/56333845/16bf4849-aa60-4569-a51a-a6a92c6b7782)
The title is the part the user sees and should show how to fix it. For example, this is the tip given by rust-analyzer

Reply:
In the latest commit, I have provided a formatted string to the `title`:
<img width="620" alt="Screenshot 2024-02-19 at 5 01 52 PM" src="https://github.com/kcl-lang/kcl/assets/134289475/a05ca2eb-a05d-4df2-9ba2-79c0c28433ce">

Is this the way to go with this?
<br>
The rust-analyzer also provides a tip along with a clickable link to the file like this:

<img width="616" alt="Screenshot 2024-02-19 at 5 06 59 PM" src="https://github.com/kcl-lang/kcl/assets/134289475/18bda02f-3e95-41c1-8bfc-b4b49452dc02">

Do we need something like this also? If yes, how to proceed?